src/commands/admin.command.ts

```diff
@@ -0,0 +1,94 @@
+import { Command, CommandMessage, Guard, Description } from "@typeit/discord";
+import { NotBot } from "../guards/NotABot.guard";
+import { Logger } from "../services/logger.service";
+import { Admin } from "../guards/Admin.guard";
+import { HACKATHON } from "../enum/hackathon.enum";
+import { Mention } from "../guards/Mention.guard";
+import { ID } from "../enum/id.enum";
+
+interface amountArgs {
+    amount: number;
+}
+
+export abstract class Admins {
+
+    logger = Logger.prototype.getInstance();
+
+    @Command("ban")
+    @Description("bans the user you @'d first")
+    @Guard(NotBot, Admin, Mention)
+    async ban(command: CommandMessage): Promise<void> {
+        this.logger.info("Sending ban");
+
+        const user = command.mentions.members.first();
+        const role = command.guild.roles.cache.get(ID.ADMIN_ID);
+
+        if (!command.member.roles.cache.has(role.id)) {
+            user.ban();
+        }
+        else {
+            this.logger.error("unable to ban " + user.displayName);
+        }
+
+        command.reply(user.displayName + " has been banned").then((messageSent) => {
```

Comment:
Can you move this to reply and DM _before_ banning? If we ban them first and there's no mutual servers we can't DM.